spec/classes/candlepin_config_spec.rb

```diff
@@ -22,6 +22,18 @@
             with_content(/^candlepin.crl.file=\/var\/lib\/candlepin\/candlepin-crl.crl$/).
             with({})
         end
+
+        it "should contain tomcat.conf" do
+          should contain_file("/etc/tomcat/tomcat.conf").
+          with_content(/^TOMCAT_CFG_LOADED=1$/).
+          with_content(/^TOMCATS_BASE=\/var\/lib\/tomcats$/).
+          with_content(/^JAVA_HOME=\/usr\/lib\/jvm\/jre$/).
+          with_content(/^CATALINA_HOME=\/usr\/share\/tomcat$/).
+          with_content(/^CATALINA_TMPDIR=\/var\/lib\/tomcats$/).
+          with_content(/^JAVA_OPTS=-xms1024m xmx4096m$/).
+          with_content(/^SECURITY_MANAGER=false$/).
+          with({})
```

Comment:
I'd indent all but the first line one level to indicate that they are a continuation of the first line.